src/components/dropdown-menu/styled.ts

```diff
@@ -0,0 +1,108 @@
+import styled, { css, createGlobalStyle } from 'styled-components';
+import { getConfig } from '../../config';
+
+const isTransportportal = getConfig().themeNap;
+
+const onMobileView = '@media (max-width: 990px)';
+
+type globalProps = {
+  dropdownOpen: boolean;
+};
+
+const GlobalStyle = createGlobalStyle<globalProps>`
+  body {
+    ${onMobileView} {
+      ${({ dropdownOpen }) =>
+        dropdownOpen &&
+        css`
+          height: 100vh;
+          overflow-y: hidden;
+
+          // For safari:
+          position: fixed;
+          left: 0;
+          right: 0;
+        `}
+    }
+  }
+`;
```

Comment:
Hvorfor er denne nødvendig?

Reply:
Dette er for å hindre scroll på bakgrunnen mens menyen er åpen i mobilmodus! Burde jeg kommentert her?